src/ui/Header/Sidebar/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { ReactComponent as ExploreIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/explore.svg";
+import { ReactComponent as HistoryIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/history.svg";
+import { ReactComponent as LibraryIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/home.svg";
+import { ReactComponent as HomeIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/library.svg";
+import { ReactComponent as LikeVideosIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/liked-videos.svg";
+import { ReactComponent as ShortsIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/shorts.svg";
+import { ReactComponent as ShowMoreIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/show-more.svg";
+import { ReactComponent as SubscriptionsIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/subscriptions.svg";
+import { ReactComponent as WatchLaterIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/watch-later.svg";
+import { ReactComponent as YourVideosIcon } from "C:/Users/mikey-pc/Workspace/mb-react-app/src/ui/Header/baricons/your-videos.svg";
+import "./index.css";
+
+const Sidebar = () => {
+  return (
+    <div className="sidebar">
+      <div className="sidebar-inner">
+        <div className="sidebar-link home">
+          <div className="icon-selector home-icon">
```

Comment:
`icon-selector` is quite confusing, since the link text is in this. when converting to `styled-components` I would use something like `LinkContent`

Reply:
I changed it to link content! I should have made it more distinct! I am wrapping the icon in its own div. It will have a personalized class so I can constrain it to one point, that way I can increase the size of my hover bar without the icons moving. Thats the idea at least. About to try it.